docs/spec/ibc/proofs.md

```diff
@@ -0,0 +1,58 @@
+## 2 Proofs
```

Comment:
I feel that this section would be much more clear if done in the context of proving validity of the message sent using IBC primitive, i.e., one way to look at this content is that it explains how we can ensure message validity in blockchain systems with instant finality. But I don't see big benefit in being abstract at this level as Tendermint is at this point only blockchain system with instant finality. We can be abstract at the level of IBC primitive by defining what we mean by validity property, and then show how we can provide in the context of Tendermint-based blockchain systems.

Reply:
This section has been changed to describe the primitives required to implement an IBC "connection", and the lifecycle of connections. Do you think it's still too abstract?